ntbs-bulk-insert/ReadMe.txt

```diff
@@ -0,0 +1,14 @@
+﻿The ntbs-bulk-insert console application is used to generate large numbers of notifications, mostly for testing how well
+the system can cope with large amounts of data.
+
+Each notification is given a minimal amount of randomised data:
+- Full name
+- Notification date between 2014 and 2017
+- Tb service
+- Nhs number (starting with 9 so that it is counted as test data by the system)
+- Optionally treatment start and completed events can be added to the system, see comment in GenerateNotifications
+- 'UniqueBulkInsert' is assigned to the notes field in ClinicalDetails to allow these records to be easily deleted
+
+The connection string for the database is pulled from appsettings.development.json in ntbs-service so to target
+a different database change the connection string here.
+
```

Comment:
How do I run it? How do I run the "alerty" version? also while we're there, perhaps we can add a flag to the run, rather than have to comment it a line out?